onnxruntime/core/protobuf/onnx-ml.proto3

```diff
@@ -506,19 +549,13 @@ message TypeProto {
     // repeated TypeProto parameters = 3;
   }
 
-  message SparseTensor { 
-    // This field MUST NOT have the value of UNDEFINED 
-    // This field MUST have a valid TensorProto.DataType value
-    // This field MUST be present for this version of the IR. 
-    int32 elem_type = 1;
-    TensorShapeProto shape = 2;
-  }
-
 
   oneof value {
     // The type of a tensor.
     Tensor tensor_type = 1;
 
+    SparseTensor sparse_tensor_type = 8;
```

Comment:
>SparseTensor sparse_tensor_type = 8; [](start = 4, length = 36)

This change adds a new compiler generated enum to this oneof statement. Add handling for that case everywhere value_case() is being handled.

Reply:
this part is just moving the line below up to here, should be ok